### ISSUE TITLE
homebrew for M1 chipset has files in /opt/homebrew

### ISSUE THREAD

**Issue by professor** [kejadlen/git-together#49]
On a new m1 laptop using Big Sur, I'm seeing this error. I installed using `brew install pivotal/tap/git-together`

```
% git-together

dyld: Library not loaded: /usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib
  Referenced from: /opt/homebrew/bin/git-together
  Reason: image not found
zsh: abort      git-together
```

It is now in `/opt/homebrew/opt/openssl@1.1/`

**Comment by professor** [comment]
Maybe it's time to move the tap? It's currently here: https://github.com/pivotal-legacy/homebrew-tap/blob/master/git-together.rb

**Comment by professor** [comment]
A work around is to install 2 homebrews on M1, one that supports brew formulas/casks that work on M1 and one that uses rosetta for everything else. 

```
 arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 alias ibrew='arch -x86_64 /usr/local/bin/brew'
ibrew install pivotal/tap/git-together
```

**Comment by kejadlen** [comment]
I don't have access to an M1 Mac, so I don't know what needs to be done here to build a release that works out of the box for both architectures.

**Comment by mikeyp** [comment]
I was able to build on an M1 and I'm using it without any issues. I'm new to rust, but some Googling shows that it should be possible to cross compile from x86. 

```
❯ file /Users/dev/.cargo/bin/git-together
/Users/dev/.cargo/bin/git-together: Mach-O 64-bit executable arm64
```

**Comment by kejadlen** [comment]
Fixed in [v0.1.0-alpha.25](https://github.com/kejadlen/git-together/releases/tag/v0.1.0-alpha.25)
